### PR TITLE
Home work4

### PR DESCRIPTION
**Pull request by hubersky**

### CODE REVIEW

**Review by hubersky — COMMENTED** [review]
Чекаю рекомендацій на покращення коду)

File: src/main/java/home_works/home_work4/first_task/Main.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package home_works.home_work4.first_task;
+
+public class Main {
+    public static void main(String[] args) {
+        Geometric circle = new Circle(7);
+        Geometric triangle = new Triangle(2, 3);
+        Geometric square = new Square(2);
+        Geometric[] geometricShape = {circle, triangle, square};
```
Comment (shaitanaleksander) [inline src/main/java/home_works/home_work4/first_task/Main.java]: name it like shapes, we can see that it geometric from the array type

File: src/main/java/home_works/home_work4/first_task/Main.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        sumGeometricShape(geometricShape);
+    }
+
+    public static void sumGeometricShape(Geometric[] arr) {
```
Comment (shaitanaleksander) [inline src/main/java/home_works/home_work4/first_task/Main.java]: arr bad name

File: src/main/java/home_works/home_work4/first_task/Main.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        float result = 0;
+        for (Geometric g : arr) {
```
Comment (shaitanaleksander) [inline src/main/java/home_works/home_work4/first_task/Main.java]: g is a bad name

File: src/main/java/home_works/home_work4/first_task/Triangle.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package home_works.home_work4.first_task;
+
+public class Triangle implements Geometric {
+    float lengthOfSide;
+    float lengthOfHeight;
```
Comment (shaitanaleksander) [inline src/main/java/home_works/home_work4/first_task/Triangle.java]: incapsulation (var need to be private )

**Review by shaitanaleksander — APPROVED** [review]

**Pull request merged**